app/src/main/java/org/jellyfin/androidtv/ui/itemhandling/BaseRowItem.java

```diff
@@ -545,6 +549,28 @@ public Drawable getBadgeImage() {
         return ContextCompat.getDrawable(TvApp.getApplication(), R.drawable.blank10x10);
     }
 
+    public Bitmap getBlurHashBitmap(double aspect) {
+        final double ASPECT_RATIO_BANNER = 5.414;
+
+        Bitmap bitmap = null;
+        if (baseItem != null && baseItem.getImageBlurHashes() != null) {
+            HashMap<String, String> blurHashMap;
+            if (aspect == ASPECT_RATIO_BANNER) {
+                blurHashMap = baseItem.getImageBlurHashes().get(org.jellyfin.apiclient.model.entities.ImageType.Banner);
+            } else if (aspect == ImageUtils.ASPECT_RATIO_16_9) {
```

Comment:
```suggestion
            } else if (aspect == ImageUtils.ASPECT_RATIO_16_9 && this.getBaseItemType() != BaseItemType.UserView) {
```
I tried this since the My Media library images are `ImageType.Primary`, but it didn't work. The blurhash loads when I ignore aspect ratio and force `ImageType.Primary` for everything, so they are there—it's just trying for the nonexistent thumb blurhash